Sources/Routing/Register/PathComponent.swift

```diff
@@ -7,9 +7,12 @@ public enum PathComponent: ExpressibleByStringLiteral {
     /// A dynamic parameter component.
     case parameter(String)
     
+    /// This route will match everything that is not in other routes
+    case anything
+    
     /// This route will match and discard any number of constant components after
     /// this anything component.
-    case anything
+    case catchall
 
     /// See `ExpressibleByStringLiteral`.
     public init(stringLiteral value: String) {
```

Comment:
Might be out of scope for this PR, but what if we added a `switch` here where if `value` is equal to `*`, we set `self` to `.anything`?

Reply:
I was thinking about that, but then you couldn't use `*` as a character in your route. Not really worth it considering we have the convenience variable anyway.